cloud/roles/postgres_connection/tasks/delete-databases.yml

```diff
@@ -0,0 +1,28 @@
+- name: Revoke privileges for the "{{ vars[ cluster_type + '_database_user'] }}" role to the "{{ vars[ cluster_type + '_database_name'] }}" database
```

Comment:
Is this necessary ? If I am not wrong, it is like cleaning up before dropping an A-bomb

Reply:
Done.